### ISSUE TITLE
Calendar doesn't scroll down on mobile

### ISSUE THREAD

**Issue by lucasmarcelli** [lucasmarcelli/mood-tracker#22]
Leaves you unable to add anything in week 4. Probably a css issue, I'll take a look. If I can't figure it out i'll let you know @OSP123

**Comment by lucasmarcelli** [comment]
![screenshot_2016-04-17-12-51-37](https://cloud.githubusercontent.com/assets/1221253/14590718/84cd86ac-04cf-11e6-9c2a-2070460b83a8.png)

**Comment by OSP123** [comment]
What was the issue? I just saw it yesterday.  

```
On Monday, April 18, 2016 4:36 AM, Lucas Marcelli <notifications@github.com> wrote:
```

 Closed #22 via 81fc64f.—

**Comment by lucasmarcelli** [comment]
I had height set to 75vh in the css. It still has the calendar cutting off
the bottom row of dates a bit but it's fine for now...
